DNN Platform/Tests/DotNetNuke.Tests.Integration/packages.config

```diff
@@ -5,8 +5,8 @@
   <package id="Moq" version="4.2.1502.0911" targetFramework="net45" allowedVersions="[4.2.1502.0911]" />
   <package id="NBuilder" version="5.0.0" targetFramework="net45" allowedVersions="[5.0.0]" />
   <package id="Newtonsoft.Json" version="10.0.3" targetFramework="net45" allowedVersions="[10.0.3]" />
-  <package id="NSubstitute" version="3.1.0" targetFramework="net45" allowedVersions="[3.1.0]" />
+  <package id="NSubstitute" version="3.1.0" targetFramework="net45" allowedVersions="[3.1.0]" requireReinstallation="true" />
```

Comment:
looks like these packages need to be [reinstalled](https://docs.microsoft.com/en-us/nuget/consume-packages/reinstalling-and-updating-packages)

Reply:
Ok, I'll get that cleaned up as well. Thanks @bdukes for the review!